lib/open_referral_transformer.rb

```diff
@@ -23,12 +29,22 @@ def initialize(args)
     @organizations_path = args[:organizations]
     @locations_path = args[:locations]
     @services_path = args[:services]
-    @output_organizations_path = "#{ENV["ROOT_PATH"]}/tmp/organizations.csv"
-    @output_locations_path = "#{ENV["ROOT_PATH"]}/tmp/locations.csv"
-    @output_services_path = "#{ENV["ROOT_PATH"]}/tmp/services.csv"
-    @output_phones_path = "#{ENV["ROOT_PATH"]}/tmp/phones.csv"
-
+    @output_dir = args[:output_dir]
```

Comment:
I would change this to be:

`@output_dir = args[:output_dir] || OUTPUT_DIRECTORY_PATH`

so that if the user doesn't provide an `output_dir` argument, the instance variable will still be set with the default.